client/web/reporter/src/components/EditorToolbar.vue

```diff
@@ -44,7 +44,7 @@
         class="text-nowrap"
```

Comment:
I now see you're using c-input-processing, my bad i didn't notice earlier.
You should use c-button-submit instead, the c-input-processing is gonna be retired soon.
If you do it same as submit button you'll see the text disappears as it should